tests/pytorch_tests/function_tests/test_tensorboard_writer.py

```diff
@@ -14,26 +14,40 @@
 # ==============================================================================
 
 
+import glob
+import os
 import unittest
+from datetime import datetime
 
-import model_compression_toolkit as mct
 import numpy as np
-from torchvision.models import mobilenet_v2
-import os
-import glob
+
+import model_compression_toolkit as mct
+from model_compression_toolkit.core import common
+from model_compression_toolkit.core.tpc_models.default_tpc.latest import generate_pytorch_tpc
+from model_compression_toolkit.core.tpc_models.default_tpc.latest import get_op_quantization_configs
+from tests.common_tests.helpers.generate_test_tp_model import generate_tp_model_with_activation_mp
+from tests.pytorch_tests.model_tests.feature_models.mixed_precision_activation_test import MixedPrecisionNet
+from tests.pytorch_tests.model_tests.feature_models.multiple_outputs_node_test import MultipleOutputsNet
 
 
 def random_datagen():
     return [np.random.random((1, 3, 224, 224))]
 
 
-class PytorchTestLogger(unittest.TestCase):
+def multiple_random_datagen():
+    return [np.random.random((1, 3, 224, 224)), np.random.random((1, 3, 224, 224))]
+
+
+class BasePytorchTestLogger(unittest.TestCase):
+
+    """
+    This is the base test of PyTorch Logger.
+    """
 
     @classmethod
     def setUpClass(cls):
-        mct.core.common.Logger.set_log_file('/tmp/')
-        model = mobilenet_v2(pretrained=True)
-        mct.pytorch_post_training_quantization(model, random_datagen, n_iter=1, analyze_similarity=True)
+        ts = datetime.now(tz=None).strftime("%d%m%Y_%H%M%S")
```

Comment:
1. why is this a class method and not a static one?
2. do you cleanup the log files aftereach run?